util/src/main/java/org/kaazing/gateway/util/GL.java

```diff
@@ -20,6 +20,8 @@
 
 public class GL {
 
+    public static final String CLUSTER_LOGGER_NAME = "ha";
```

Comment:
@mjolie, suggest moving this to: ClusterContext class.  GL class is an antipattern for logging (inefficient because every call looks up the logger).  Sorry for miss directing you